sources/pam.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Odyssey.
+ *
+ * Scalable PostgreSQL connection pooler.
+*/
+
+#include <security/pam_appl.h>
+#include <kiwi.h>
+
+struct auth_request {
+    char *password;
+};
+
+
+static int
+pam_conversation(int msgc,
```

Comment:
Could you please fix styling in this file until PR with auto-format will be merged